l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -288,6 +288,11 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
 
         address hyperchain = getHyperchain(_request.chainId);
 
+        require(
```

Comment:
Is this really needed? If we can avoid smart contract changes it would be perfect!

Reply:
This "require" is after using this address with a shared bridge interface. I can mock it, but logically it should be as I proposed. I can change it back if you would like to keep it this way.